assets/styles/homepage.scss

```diff
@@ -268,6 +272,7 @@ h2 {
 // Contact section extra padding so scrollspy works
 .section-end {
   @include media ($medium-screen) {
+    padding-top: 13rem;
```

Comment:
I would not add this, to keep it consistent with the others. Let's change it back.